pom.xml

```diff
@@ -524,8 +532,8 @@
                         <configuration>
                           <rules>
                             <requireMavenVersion>
-                              <version>[3.2.5,)</version>
-                              <message>Building ESAPI 2.x now requires Maven 3.2.5 or later.</message>
+                              <version>[3.8.0,)</version>
```

Comment:
Minimum Maven version is now 3.8.0 to support plugin updates.
  I don't recall which ones exactly, but if we need to track that down we can.